spec/services/proofing/aamva/request/verification_request_spec.rb

```diff
@@ -89,6 +89,87 @@
       )
     end
 
+    it 'includes height if it is present' do
+      applicant.height = '63'
+      expect(subject.body).to include(
+        '<aa:PersonHeightMeasure>63</aa:PersonHeightMeasure>',
+      )
+    end
+
+    it 'includes weight if it is present' do
+      applicant.weight = 190
+      expect(subject.body).to include(
+        '<aa:PersonWeightMeasure>190</aa:PersonWeightMeasure>',
+      )
+    end
+
+    it 'includes eye_color if it is present' do
+      applicant.eye_color = 'blu'
+      expect(subject.body).to include(
+        '<aa:PersonEyeColorCode>blu</aa:PersonEyeColorCode>',
+      )
+    end
+
+    it 'includes name_suffix if it is present' do
+      applicant.name_suffix = 'JR'
+      expect(subject.body).to include(
+        '<nc:PersonNameSuffixText>JR</nc:PersonNameSuffixText>',
+      )
+    end
+
+    context '#sex' do
+      context 'when the sex is male' do
+        it 'sends a sex code value of 1' do
+          applicant.sex = 'male'
+          expect(subject.body).to include(
+            '<aa:PersonSexCode>1</aa:PersonSexCode>',
+          )
+        end
+      end
+
+      context 'when the sex is female' do
+        it 'sends a sex code value of 2' do
+          applicant.sex = 'female'
+          expect(subject.body).to include(
+            '<aa:PersonSexCode>2</aa:PersonSexCode>',
+          )
+        end
+      end
+
+      context 'when the sex is blank' do
+        it 'does not send a sex code value' do
+          applicant.sex = nil
+          expect(subject.body).to_not include('<aa:PersonSexCode>')
+        end
+      end
```

Comment:
(Also noting that this test specifically covers the concern I had erroneously raised.)